cypress/e2e/awx/resources/inventoryHost.cy.ts

```diff
@@ -1,145 +1,168 @@
 import { Inventory } from '../../../../frontend/awx/interfaces/Inventory';
-import { InventorySource } from '../../../../frontend/awx/interfaces/InventorySource';
 import { Organization } from '../../../../frontend/awx/interfaces/Organization';
-import { Project } from '../../../../frontend/awx/interfaces/Project';
 import { Credential } from '../../../../frontend/awx/interfaces/Credential';
 import { ExecutionEnvironment } from '../../../../frontend/awx/interfaces/ExecutionEnvironment';
 
 describe('Inventory Host Tab Tests', () => {
   let organization: Organization;
-  let project: Project;
   let inventory: Inventory;
-  let inventorySource: InventorySource;
   let machineCredential: Credential;
   let executionEnvironment: ExecutionEnvironment;
+  const kinds: Array<'' | 'smart' | 'constructed'> = ['', 'smart', 'constructed'];
 
   before(() => {
     cy.awxLogin();
   });
 
   afterEach(() => {
-    cy.deleteAwxProject(project, { failOnStatusCode: false });
-    cy.deleteAwxInventorySource(inventorySource, { failOnStatusCode: false });
     cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
     cy.deleteAwxCredential(machineCredential, { failOnStatusCode: false });
     cy.deleteAwxExecutionEnvironment(executionEnvironment, { failOnStatusCode: false });
     cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
   });
 
   describe('Inventory Host List Tests', () => {
-    beforeEach(() => {
-      cy.createAwxOrganization().then((org) => {
-        organization = org;
-        cy.createAWXCredential({
-          kind: 'machine',
-          organization: organization.id,
-          credential_type: 1,
-        }).then((cred) => {
-          machineCredential = cred;
-        });
-        cy.createAwxExecutionEnvironment({ organization: organization.id }).then((ee) => {
-          executionEnvironment = ee;
-          cy.createInventoryHost(organization).then((result) => {
-            const { inventory, host } = result;
-            cy.log('Inventory', inventory); //remove this once it is written into the tests below
-            cy.log('Host', host); //remove this once it is written into the tests below
-            cy.log('EE', ee); //remove this once it is written into the tests below
-            cy.log('Machine Cred', machineCredential); //remove this once it is written into the tests below
+    kinds.forEach((kind) => {
```

Comment:
Move this loops please above the describe, because now it is generating whole content of the describe multiple times including beforeEach. For it, its ok, its can be duplicated and it will work as expected, but beforeEach should be there only once.